xml/src/main/java/org/ehcache/config/xml/XmlConfiguration.java

```diff
@@ -296,6 +296,10 @@ public URL getURL() {
         .usingEvictionPrioritizer(getInstanceOfName(cacheTemplate.evictionPrioritizer(), defaultClassLoader, EvictionPrioritizer.class, Eviction.Prioritizer.class))
         .evitionVeto(getInstanceOfName(cacheTemplate.evictionVeto(), defaultClassLoader, EvictionVeto.class))
         .withExpiry(getExpiry(defaultClassLoader, parsedExpiry));
+    final String loaderWriter = cacheTemplate.loaderWriter();
```

Comment:
This should be done the same way it is done in `org.ehcache.config.xml.XmlConfiguration#parseConfiguration`